### ISSUE TITLE
Incorrect variable types in routelink file result in inconsistent model behavior

### ISSUE THREAD

**Issue by kafitzgerald** [NCAR/wrf_hydro_nwm_public#40]
## Expected Behavior
If variable types are important, the model should check for them and notify the user

## Current Behavior
The model is failing testing because of inconsistent behavior resulting from incorrect variable types in the new routelink file

## Possible Solution
Add in checks for appropriate variable types and roll back testing to old model domain for the time being

**Comment by jmills-ncar** [comment]
Katelyn FitzGerald [6 days ago]
Basically the new Route_Link.nc file that Joe (I think) created is causing model testing to fail, because of some weird model behavior when there are incorrect types in the Route_Link.nc file.

**Comment by jmills-ncar** [comment]
Yes, I edited the route_link for NWM to remove the errant gage that was present, and then re-saved the file. I did not pay attention to variable types because I thought that was handled by the netCDF standard. Is it not?

**Comment by rcabell** [comment]
This should be rolled up into an Input/Configuration Validation meta-issue. Tagging pending that issue being opened.
